test/components/kubernetes/namespaces_labels_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws aws_edge packet aks baremetal
+// +build e2e
+
+package kubernetes_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kinvolk/lokomotive/internal"
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+func TestAllNamespacesHaveLabels(t *testing.T) {
+	client := testutil.CreateKubeClient(t)
+
+	namespaces, err := client.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		t.Fatalf("expected a list of namespaces")
+	}
+
+	for _, namespace := range namespaces.Items {
+		name := namespace.ObjectMeta.Name
+		labels := namespace.ObjectMeta.Labels
+
+		t.Logf("testing namespace %q", name)
+		// Do not consider the namespace as its generated during the test of metadata access.
+		if strings.Contains(name, "metadata-access-test-") {
```

Comment:
:see_no_evil: Let's use code from https://github.com/kinvolk/lokomotive/pull/704#pullrequestreview-464158079 for it.

Reply:
cherry picked the commit :)